hole/jacobi-symbol.go

```diff
@@ -31,41 +30,34 @@ func randomPrime(bits int) *big.Int {
 }
 
 func jacobiSymbol() ([]string, string) {
-	const tests = 20
-	type Test struct {
+	const mult = 20
```

Comment:
What does mult mean in this context?

Reply:
> What does mult mean in this context?

I guess I had the word "multiplicity" (which I don't know if such word exists) or "multiplier" in mind.